modules/nextflow/src/test/groovy/nextflow/datasource/SraExplorerTest.groovy

```diff
@@ -112,6 +112,25 @@ class SraExplorerTest extends Specification {
+
+ */
+
+        given:
+        def slurper = new SraExplorer()
+        def exp = "study_accession\tsecondary_study_accession\tsample_accession\tsecondary_sample_accession\texperiment_accession\trun_accession\tsubmission_accession\ttax_id\tscientific_name\tinstrument_platform\tinstrument_model\tlibrary_name\tnominal_length\tlibrary_layout\tlibrary_strategy\tlibrary_source\tlibrary_selection\tread_count\tbase_count\tcenter_name\tfirst_public\tlast_updated\texperiment_title\tstudy_title\tstudy_alias\texperiment_alias\trun_alias\tfastq_bytes\tfastq_md5\tfastq_ftp\tfastq_aspera\tfastq_galaxy\tsubmitted_bytes\tsubmitted_md5\tsubmitted_ftp\tsubmitted_aspera\tsubmitted_galaxy\tsubmitted_format\tsra_bytes\tsra_md5\tsra_ftp\tsra_aspera\tsra_galaxy\tcram_index_ftp\tcram_index_aspera\tcram_index_galaxy\tsample_alias\tbroker_name\tsample_title\tnominal_sdev\tfirst_created\n" +
+                "PRJNA30709\tSRP007461\tSAMN00634070\tSRS214591\tSRX082565\tSRR307897\tSRA039973\t9606\tHomo sapiens\tILLUMINA\tIllumina Genome Analyzer II\tGSM758559: CshlLong_RnaSeq_GM12878_cell_longPolyA\t\tPAIRED\tRNA-Seq\tTRANSCRIPTOMIC\tcDNA\t32214376\t4896585152\tGEO\t2011-07-28\t2015-06-19\tIllumina Genome Analyzer II sequencing; GSM758559: CshlLong_RnaSeq_GM12878_cell_longPolyA\tGSE30567: Long RNA-seq from ENCODE/Cold Spring Harbor Lab\tGSE30567\tGSM758559: lab_RnaSeq_GM12878_cell_longPolyA\tGSM758559_1\t2392496788;2436856506\tcdeb9d2450ca0bfc678968ac33f551a6;a1e9011d47d5b0462b9d50402a92dc6d\tftp.sra.ebi.ac.uk/vol1/fastq/SRR307/SRR307897/SRR307897_1.fastq.gz;ftp.sra.ebi.ac.uk/vol1/fastq/SRR307/SRR307897/SRR307897_2.fastq.gz\tfasp.sra.ebi.ac.uk:/vol1/fastq/SRR307/SRR307897/SRR307897_1.fastq.gz;fasp.sra.ebi.ac.uk:/vol1/fastq/SRR307/SRR307897/SRR307897_2.fastq.gz\tftp.sra.ebi.ac.uk/vol1/fastq/SRR307/SRR307897/SRR307897_1.fastq.gz;ftp.sra.ebi.ac.uk/vol1/fastq/SRR307/SRR307897/SRR307897_2.fastq.gz\t\t\t\t\t\t\t3499798854\t2ee356e58e9765d6b26bde4b119b99be\tftp.sra.ebi.ac.uk/vol1/srr/SRR307/SRR307897\tfasp.sra.ebi.ac.uk:/vol1/srr/SRR307/SRR307897\tftp.sra.ebi.ac.uk/vol1/srr/SRR307/SRR307897\t\t\t\tGSM758559\t\tCSHL_RnaSeq_GM12878_cell_longPolyA (superseded by GSE86658)\t\t2011-07-28\n"
+
+        when:
+        def result = slurper.parseCsv(exp)
+        then:
+        result['study_accession'] == 'PRJNA30709'
```

Comment:
Check more fields, above all the multi-values attributes that have to be returned as List ie. read file urls.

Reply:
Added here: 517a239 
I checked last and first fields and several multi-values attributes, should we test all of them